scripts/Makefile

```diff
@@ -14,9 +14,10 @@ build : ./Dockerfile ./build_wheels_linux.sh
 wheels : ./Dockerfile ./build_wheels_linux.sh build
 	mkdir -p $(HOME)/wheelhouse
 	# Generate version.
-	docker run -e PYPI_PASSWORD=$(PYPI_PASSWORD) \
-		$(LABEL)
-
+	$(eval CONTAINER = $(shell docker run -it --detach $(LABEL)))
```

Comment:
This probably will not result in `CMD` statement succeeding: it will terminate immediately. I am guessing this is to get the container id and then use `docker exec` to build the wheels.